### PR TITLE
[plugin.video.fosdem] 0.0.8

### PR DESCRIPTION
**Pull request by jelly**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.fosdem/addon.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.fosdem" name="FOSDEM Videos" version="0.0.6" provider-name="Jelle van der Waa">
+<addon id="plugin.video.fosdem" name="FOSDEM Videos" version="0.0.7" provider-name="Jelle van der Waa">
```
Comment (basrieter) [inline plugin.video.fosdem/addon.xml]: Is it correct that this is version 0.0.7 and the Krypton one is 0.0.8? That won't work. Most common is to version the Matrix version with "0.0.8+matrix.1" if the Krypton version is 0.0.8.

File: plugin.video.fosdem/resources/lib/addon.py [hunk 2]
```diff
@@ -30,8 +28,15 @@ def years():
...
+def get_setting_int(name):
+    val = getSetting(plugin.handle, name)
+    if not val:
+        val = '0'
+    return int(val)
+
+
 def get_format():
-    return FORMATS[addon.getSettingInt('format')]
+    return FORMATS[get_setting_int('format')]
```
Comment (basrieter) [inline plugin.video.fosdem/resources/lib/addon.py]: The is an `Addon.getSettingInt()`. See https://romanvm.github.io/Kodistubs/_autosummary/xbmcaddon.html

File: plugin.video.fosdem/resources/settings.xml [hunk 3]
```diff
@@ -1,21 +1,4 @@
...
-							<option label="webm">1</option>
-						</options>
-					</constraints>
-					<control type="spinner" format="string"/>
-				</setting>
-
-			</group>
-		</category>
-	</section>
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```
Comment (basrieter) [inline plugin.video.fosdem/resources/settings.xml]: Why are you reverting back to the deprecated settings format?

**Review by basrieter — CHANGES_REQUESTED** [review]
This PR look weird? Als if you are reverting Python 3 stuff, instead of introducing them?

**Comment by jelly** [comment]
> This PR look weird? Als if you are reverting Python 3 stuff, instead of introducing them?

I guess I reverted that on my own branch because it was needed for the older supported kodi versions. I should I guess make branches for them so I don't move incompatible changes to older versions.

**Comment by basrieter** [comment]
I guess so. If you fix up the Matrix PR, I can review it and then the Leia/Krypton (https://github.com/xbmc/repo-plugins/pull/3893) version can be reviewed. That is the correct order.

**Review by basrieter — APPROVED** [review]

**Pull request merged**